_posts/2018-12-06.Rust-1.31.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: "Announcing Rust 1.31"
```

Comment:
We had talked in the core team about more clearly distinguishing this from a normal announcement.

Maybe "Announcing Rust 2018 (1.31)" or some such?

Reply:
I've made a tweak, let me know if you want a further one